tocrify/api/hocr.py

```diff
@@ -109,6 +113,7 @@ def _fromfile(self, path):
         Reads in hOCR from a given file source.
         :param str path: Path to a hOCR document.
         """
+        self.logger.info("Reading %s" % str(path))
```

Comment:
The `str` seems unnecessary, string interpolation should already do that.

Also with python logging you can do lazy string interpolation, like so:

```python
self.logger.info("Reading %s", path)
```

That way, the log message is only generated if that log level is actually active. Not a big deal here but is good practice when logging complex objects or big strings.